website/docs/guides/custom-cicd-pipelines.md

```diff
@@ -246,6 +246,8 @@ The YAML file will look pretty similar to our earlier job, but there is a new se
 
 It’s worth noting that we changed the `on:` section to now run **only** when there are pushes to a branch named `main` (i.e. a PR is merged). Have a look through [GitHub’s docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) on these filters for additional use cases.
 
+For the value of `DBT_JOB_BRANCH`, refer to the [GitHub documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs) for information about `github` context property names and their use cases. 
```

Comment:
> For the value of `DBT_JOB_BRANCH`

I don't think they need to add a value for  `${{ github.ref_name }}` it should be generated automatically because it triggers the workflow. Unless I'm misunderstanding something?